### DISCUSSION TITLE
Windows: Cursor shows Loading Indicator every like 15 seconds

### DISCUSSION THREAD

**Discussion by stefan1495** [timmo001/system-bridge#2856] [Help] [answered]
Hello,
not sure if it´s only my system or actually an issue related to every windows user, but like every 15 seconds i get a loading circle on my cursor when i have System Bridge installed. Once uninstalled, the Loading Indicator is gone. This may seem like a minor thing, but it keeps me from using this software on a daily basis for my Home Assistant Setup :(

Is there anything i can do about it?

**Comment by timmo001** [comment]
I actually dont see this myself. I would guess the reason is because of the writing to a sqlite database every 30 seconds

**Reply by stefan1495** [reply]
Is there actually anything I can do about it? I tried changing the process to high priority. Can it be caused by e.g. many Drives? Got 6 Drives shown on my Explorer - I would assume it's not related to missing computing power as I got a ryzen 7 5800x3d + 32 gb DDR4 ram.

**Reply by timmo001** [reply, marked as answer]
I have a system running probably more than that and don't see the cursor spinning. I guess you could edit your mouse settings to use the normal cursor instead

**Comment by aderusha** [comment]
I'm seeing the same behavior here: Win 11, 128GB DDR5, i9-13900k.  There should be plenty enough resources and I don't feel that editing mouse animations to hide this behavior is a great solution.
